package.json

```diff
@@ -26,12 +26,13 @@
     "@angular/platform-browser": "~8.2.5",
     "@angular/platform-browser-dynamic": "~8.2.5",
     "@angular/router": "~8.2.5",
-    "@ng-select/ng-select": "^3.0.7",
     "@ctrl/ngx-codemirror": "^2.1.0",
+    "@ng-select/ng-select": "^3.0.7",
     "@ngx-progressbar/core": "^5.3.2",
     "angular-tree-component": "~8.4.0",
     "animate.css": "^3.7.2",
     "chart.js": "~2.8.0",
+    "chartjs-plugin-zoom": "^0.7.4",
```

Comment:
how much build size has increased with it?

Reply:
Build size is very much the same